--- kmplot-15.12.2/kmplot/view.cpp~	2016-01-08 23:07:13.000000000 +0000
+++ kmplot-15.12.2/kmplot/view.cpp	2016-03-03 17:17:19.896680700 +0000
@@ -476,7 +476,7 @@
 	double x = pixel.x();
 	double y = pixel.y();
 	
-	if ( isnan(x) )
+	if ( std::isnan(x) )
 	{
 		xclipflg = true;
 		x = pixelIfNaN.x();
@@ -503,7 +503,7 @@
 			x = m_clipRect.right();
 	}
 	
-	if ( isnan(y) )
+	if ( std::isnan(y) )
 	{
 		yclipflg = true;
 		y = pixelIfNaN.y();
